build.gradle

```diff
@@ -152,7 +152,7 @@ allprojects {
     description = 'Executes non-destructive smoke tests against a running instance'
   }
 
-  task functional(dependsOn: [':codeceptFunctional', ':pa11y']) {
+  task functional(dependsOn: [':codeceptFunctional', ':pa11y', 'functional1']) {
```

Comment:
There is no functional1 task in the gradle build? why is it required?

Reply:
This was something missed to remove This has been removed now in the latest push